src/test/java/com/thomashan/coup/action/state/WaitingMainActionStateTestCases.java

```diff
@@ -6,25 +6,25 @@
 
 import static com.thomashan.coup.PlayerBuilder.build;
 
-abstract class WaitingMainActionStateTestCases {
+public class WaitingMainActionStateTestCases {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Each class should declare at least one constructor](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4507736)